src/InputWithLabel.js

```diff
@@ -0,0 +1,25 @@
+import React, { useRef, useEffect } from 'react';
+
+const InputWithLabel = (props) => {
```

Comment:
Let's be consistent with our coding pattern. Throughout your app you've been destructuring your props however here you are passing the prop object. Either choice is fine. Being consistent is good practice.